### DISCUSSION TITLE
webui.cmd reinstalls dependencies on each run

### DISCUSSION THREAD

**Discussion by ghost** [hlky/stable-diffusion#257] [Q&A] [answered]
Just wondering if this was normal behavior, I don't recall it doing this from a version about a week ago.

I get this in the terminal:

Installing pip dependencies: | Ran pip subprocess with arguments:
['C:\\Users\\MYUSERNAME\\.conda\\envs\\ldm\\python.exe', '-m', 'pip', 'install', '-U', '-r', 'D:\\Documents\\Programs\\stable-diffusion-main\\condaenv.iofrzd9q.requirements.txt']
Pip subprocess output:
Requirement already satisfied: albumentations==0.4.3

...etc etc, 200 more dependencies listed. Any idea of what I can edit to stop this?

**Comment by ghost** [comment, marked as answer]
I figured out how to stop all the hullabaloo from happening before relauncher.py is called. Comment out the following lines in webui.cmd like so:

:: call conda env create -n "%conda_env_name%" -f environment.yaml
:: call conda env update -n "%conda_env_name%" --file environment.yaml --prune

Now I am sure these are put here to ensure everything is in its proper place before running, but it appears to me this is fine to do after the first run.  If you do a git pull you probably ought to let it do this again once to make sure it's synced up to any changes made to environment.yaml.

**Reply by ZeroCool22** [reply]
A big thx for this man!

![giphy](https://user-images.githubusercontent.com/13344308/188337860-0f3e0ad2-c49c-4d52-bb08-e8c5eb917ea1.gif)
